bin/pot-dist-references.js

```diff
@@ -10,6 +10,10 @@ const additionalReferences = [
 		re: /^#: assets\/setup-wizard\//gm,
 		reference: '#: assets/dist/setup-wizard/index.js:1',
 	},
+	{
+		re: /^#: assets\/data-port\/\//gm,
```

Comment:
It's not working because there's an extra slash in the RegExp. The correct would be:

`re: /^#: assets\/data-port\//gm,`

Reply:
Good catch 7ae78a4e68ea427e286db968c1ddf09dfa7b0f3c